packages/mux-uploader-react/src/index.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import type { CSSProperties } from 'react';
+import '@mux/mux-uploader';
+import MuxUploaderDrop from './mux-uploader-drop';
+import type MuxUploaderElement from '@mux/mux-uploader';
+import { toNativeProps } from './common/utils';
+import { useRef } from 'react';
+import { useCombinedRefs } from './useCombinedRefs';
+
+export type MuxUploaderRefAttributes = MuxUploaderElement;
+
+export type MuxUploaderProps = {
+  url: string;
+  id?: string;
+  type?: string;
+  uploadInProgress?: boolean;
+  uploadError?: boolean;
+  status?: boolean;
+  disableDrop?: boolean;
+  style?: CSSProperties;
+  children?: React.ReactNode;
+};
+
+const MuxUploaderInternal = React.forwardRef<MuxUploaderRefAttributes, MuxUploaderProps>(
+  ({ children, ...props }, ref) => {
+    return React.createElement('mux-uploader', toNativeProps({ ...props, ref }), children);
+  }
+);
+
+const useUploader = (
```

Comment:
nitpick(non-blocker): This is currently not "doing any work" but will likely be useful when event callbacks are added. Consider removing.